### PR TITLE
Add `IMPORT SCHEMA CONFIGURATION` syntax to DistSQL.

### PR DESCRIPTION
**Pull request by yx9o**
For #14807, add `IMPORT SCHEMA CONFIGURATION` syntax to DistSQL.

Changes proposed in this pull request:
Add `IMPORT SCHEMA CONFIGURATION` syntax.
Add unit test.

38eee504ab3b3e2447e82a9f7433dc5
image
image

### CODE REVIEW

**Comment by RaigorJiang** [comment]
Hi @yx9o , some CI tasks failed, please check them.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/ImportSchemaConfigurationHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        result.setJdbcUrl(String.format("jdbc:mysql://127.0.0.1:3306/%s?serverTimezone=UTC&useSSL=false", dbName));
+        result.setUsername("root");
+        result.setPassword("");
+        result.setConnectionTimeout(30000L);
+        result.setIdleTimeout(60000L);
+        result.setMaxLifetime(1800000L);
+        result.setMaximumPoolSize(50);
+        result.setMinimumIdle(1);
+        return result;
+    }
```
Comment (lanchengx) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/ImportSchemaConfigurationHandlerTest.java:124]: @yx9o Using h2 database could solve the exception of ci, please refer https://github.com/apache/shardingsphere/pull/16147/files#diff-48ebddcde35067de8df4e9f553c535a82936f1da029378744ab1d6ced60a502c

Comment (yx9o) [reply shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/ImportSchemaConfigurationHandlerTest.java:124]: > @yx9o Using h2 database could solve the exception of ci, please refer https://github.com/apache/shardingsphere/pull/16147/files#diff-48ebddcde35067de8df4e9f553c535a82936f1da029378744ab1d6ced60a502c

Thank you.

**Review by RaigorJiang — APPROVED** [review]

**Comment by RaigorJiang** [comment]
@yx9o Thank you, good job!

**Pull request merged**